[macOS] [3.2] Build fails - 'free' undeclared

<!-- Please search existing issues for potential duplicates before filing yours:
https://github.com/godotengine/godot/issues?q=is%3Aissue
-->

**Godot version:**

3.2 branch 30c69c296b66c353fe49a56b91d91b0207689114


**OS/device including version:**

macOS 10.14.6

```
$ clang --version
Apple clang version 11.0.0 (clang-1100.0.33.17)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
```

scons command: `scons target=debug dev=yes tools=yes --jobs=$(sysctl -n hw.logicalcpu)`

**Issue description:**

Compiler error:

```
godot3/modules/tinyexr/image_saver_tinyexr.cpp:279: error: use of undeclared identifier 'free'; did you mean 'DefaultAllocator::free'?
modules/tinyexr/image_saver_tinyexr.cpp:279:3: error: use of undeclared identifier 'free'; did you mean 'DefaultAllocator::free'?
                free(mem);
                ^~~~
                DefaultAllocator::free
./core/os/memory.h:66:29: note: 'DefaultAllocator::free' declared here
        _FORCE_INLINE_ static void free(void *p_ptr) { Memory::free_static(p_ptr, false); }
                                   ^
```

Introduced in [ Backport EXR compression support from master ](https://github.com/godotengine/godot/commit/7241139356e78e275e5f11c644e55e1c1963d021) (git hash 77438f7a453beac5d4a31c5690210a41e9187cec)

--- a/modules/tinyexr/image_saver_tinyexr.cpp
+++ b/modules/tinyexr/image_saver_tinyexr.cpp
@@ -33,6 +33,8 @@
 
 #include "thirdparty/tinyexr/tinyexr.h"
 
+#include <cstdlib>
+
 static bool is_supported_format(Image::Format p_format) {
 	// This is checked before anything else.
 	// Mostly uncompressed formats are considered.

